README.md

```diff
@@ -1 +1,66 @@
-# Portfolio-Divyansh
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# This project is a personal portfolio.
+
+> It is a personal portfolio with all the proffesional details about me.
+
```

Comment:
[OPTIONAL]
- [ ] You've done a great job implementing all the changes requested by the reviewer. Now it's time to show it off. Let's update our README with a screen of the app. Cheer 🥇